src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -293,11 +293,14 @@ void AtlasEngine::_drawCursor()
     // At 150% scale lineWidth thus needs to be 1.33333... because at a zoom scale of 1.5 this results in a 2px wide line.
     const auto lineWidth = std::max(1.0f, static_cast<float>((_r.dpi + USER_DEFAULT_SCREEN_DPI / 2) / USER_DEFAULT_SCREEN_DPI * USER_DEFAULT_SCREEN_DPI) / static_cast<float>(_r.dpi));
     const auto cursorType = static_cast<CursorType>(_r.cursorOptions.cursorType);
-    D2D1_RECT_F rect;
-    rect.left = 0.0f;
-    rect.top = 0.0f;
-    rect.right = _r.cellSizeDIP.x;
-    rect.bottom = _r.cellSizeDIP.y;
+
+    D2D1_RECT_F clip;
+    clip.left = 0.0f;
+    clip.top = 0.0f;
+    clip.right = _r.cellSizeDIP.x;
+    clip.bottom = _r.cellSizeDIP.y;
+
+    auto rect = clip;

```

Comment:
Why do we need clip? I mean, I see that we call `PushAxisAlignedClip()` down below, but it would be nice to add a comment here explaining the purpose of clip and what it fixes, exactly. More just so that anybody that comes along understands this code better.

Reply:
Let me know if the comment I just added is sufficient. 🙂